### DISCUSSION TITLE
Change Token BackgroundSkeleton

### DISCUSSION THREAD

**Discussion by vitorvlopes** [Telefonica/mistica-design#1832] [Proposals]
### Goal

Change token BackgroundSkeleton from grey4 (#949494) to grey3(#DDDDDD).

### Context

The current color of BackgroundSkeleton (grey4) does not meet minimal accessibility requirements. Based on research conducted by the Vivo Accessibility team, grey3 has been identified as a better alternative.

### Impact

This change will enhance contrast, improving visualization for all users, including individuals with visual disabilities.

### Discussion Questions

The BackgroundSkeleton token also affects the Background color of the Fallback Image. Are there other components that might be impacted by this change?

### Additional considerations

The Token BackgroundSkeleton color is also changing the Background color of Fallback Image

### Do you have a visual proposal?

https://www.figma.com/design/SakPOIOx0axv3iIJRHoEzh/Evolu%C3%A7%C3%A3o-Token-Border?node-id=1141-12325&t=b1kpgb3BBasfTYWA-1

**Comment by aweell** [comment]
Hi @vitorvlopes,

Sure, we can change the `backgroundSkeleton` token from `grey4` to `grey3`.

I'll create an issue from this discussion.

For this type of changes, just to let you know that we've create [a guide](https://github.com/Telefonica/mistica-design/blob/production/guides/colors-contribution.md) that explain the whole process of updating a token in any skin in the mistica-design repo, so either:
* you can assign yourself and create a PR with the changes adding @yceballost or @aweell as reviewers
* we can add it to our backlog and update it

> The BackgroundSkeleton token also affects the Background color of the Fallback Image. Are there other components that might be impacted by this change?

It shouldn't, **image** and **skeleton** are the only two components that are currently using `backgroundSkeleton` token
